cli/src/test/integration-tests/help.sh

```diff
@@ -0,0 +1,4 @@
+#--- Body ---
+set -e
+echo "Running help integration test"
+ide help
```

Comment:
just like JUnit it is best practice to assert the results.
This test only verifies that `ide help` will not cause an error but if the help output is totally messed up, it will still be green.
Shouldn't we verify that some keywords are found in the output?

Reply:
I've added an `assertThat` function and some tests to the help test now.